mungegithub/mungers/approvers/owners.go

```diff
@@ -200,14 +228,13 @@ func (ap Approvers) UnapprovedFiles() sets.String {
 // UnapprovedFiles returns owners files that still need approval
 func (ap Approvers) GetFiles() []File {
 	allOwnersFiles := []File{}
+	status := ap.Owners.GetApprovalStatus(ap.Approvers)
```

Comment:
then this becomes

`status := ap.getApprovalStatus()`  

I think it's cleaner